package.json

```diff
@@ -44,6 +44,7 @@
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^28.1.2",
+    "babel-plugin-dynamic-import-node": "^2.0.0",
```

Comment:
Move this dependency to `project-utils`, that's where that dependency is being used.